SpatialGDK/Source/SpatialGDK/Public/Utils/WorkerVersionCheck.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "improbable/c_worker.h"
+
+#define WORKER_SDK_VERSION "14.8.0"
+
+constexpr bool StringsEqual(char const* A, char const* B)
+{
+	return *A == *B && (*A == '\0' || StringsEqual(A + 1, B + 1));
+}
+
+// Check if the Worker SDK is compatible with the current version of Unreal Engine
```

Comment:
Similarly here, I would rather say current version of the GDK, since the engine is kind of agnostic to the worker SDK version.